modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -8,39 +8,179 @@
+                  "enabled": true,
+                  "name": "var-lib-containers.mount"
+                }
+              ]
+            }
+          }   
+[...]
+----
++
+[NOTE]
+====
+If the `.spec.clusters.nodes.ignitionConfigOverride` field does not exist, create it.  
+====
+
+.Verification
+
+. During or after installation, verify on the hub cluster that the `BareMetalHost` object shows the annotation.
++
+--
+[source,terminal]
+----
+$ oc get bmh -n my-sno-ns my-sno -ojson | jq '.metadata.annotations["bmac.agent-install.openshift.io/ignition-config-overrides"]
+----
+
+.Example output
+[source,terminal]
+----
+"{\"ignition\":{\"version\":\"3.2.0\"},\"storage\":{\"disks\":[{\"device\":\"/dev/disk/by-id/wwn-0x6b07b250ebb9d0002a33509f24af1f62\",\"partitions\":[{\"label\":\"var-lib-containers\",\"sizeMiB\":0,\"startMiB\":250000}],\"wipeTable\":false}],\"filesystems\":[{\"device\":\"/dev/disk/by-partlabel/var-lib-containers\",\"format\":\"xfs\",\"mountOptions\":[\"defaults\",\"prjquota\"],\"path\":\"/var/lib/containers\",\"wipeFilesystem\":true}]},\"systemd\":{\"units\":[{\"contents\":\"# Generated by Butane\\n[Unit]\\nRequires=systemd-fsck@dev-disk-by\\\\x2dpartlabel-var\\\\x2dlib\\\\x2dcontainers.service\\nAfter=systemd-fsck@dev-disk-by\\\\x2dpartlabel-var\\\\x2dlib\\\\x2dcontainers.service\\n\\n[Mount]\\nWhere=/var/lib/containers\\nWhat=/dev/disk/by-partlabel/var-lib-containers\\nType=xfs\\nOptions=defaults,prjquota\\n\\n[Install]\\nRequiredBy=local-fs.target\",\"enabled\":true,\"name\":\"var-lib-containers.mount\"}]}}"
 ----
-<1> This setting depends on the hardware. The setting can be a serial number or device name. The value must match the value set for `rootDeviceHints`.
-<2> The minimum value for `size` is 102500 MiB.
-<3> The minimum value for `start` is 25000 MiB. The total value of `size` and `start` must not exceed the disk size, or the installation will fail.
+--
 
-. Save the `SiteConfig` CR and push it to the site configuration repo.
+. After installation, check the {sno} disk status:
++
+--
+[source,terminal]
+----
+# lsblk
+----
 
-The {ztp} pipeline provisions the cluster using the `SiteConfig` CR and configures the disk partition.
+.Example output
+[source,terminal]
+----
+NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
+sda      8:0    0 446.6G  0 disk
+├─sda1   8:1    0     1M  0 part
+├─sda2   8:2    0   127M  0 part
+├─sda3   8:3    0   384M  0 part /boot
+├─sda4   8:4    0 243.6G  0 part /var
+│                                /sysroot/ostree/deploy/rhcos/var
+│                                /usr
+│                                /etc
+│                                /
+│                                /sysroot
+└─sda5   8:5    0 202.5G  0 part /var/lib/containers
+----
+
+
+[source,terminal]
+----
+# df -h
```

Comment:
Here's the other change that should be made if this is a command:
```suggestion
$ df -h
```

Reply:
I picked this section up from another PR bit as I checked this looks like there is some missing guidance. These commands are executed on the SNO node so you need to log in to the node to run both commands